tests/test_parsers.py

```diff
@@ -156,6 +156,32 @@ def test_distributed_availability(self):
             OrderedDict([('1', 1), ('2', 4)])
         )
 
+    def test_real_time_payment_credit(self):
+        lines = [
+            '16,158,4346722,,0,0/',
+            '88, ACC NUM:=55668;',
+            '88, SENDER BNK:=PNC; ',
+            '88, SENDER ID:=0096;',
+            '88, SENDER ACC NUM:=4990109778;',
+            '88, ORG ID:=null;',
+            '88, ORG:=PI ARCHITECTS ;',
+            '88, ORG ADDRESS:=6010 Main Sy AUSTIN TX US 78731-4297 ;',
+            '88, ORIG FI ID:=00000096; ',
+            '88, ORG FI ADDRESS:=null;',
+            '88, BNF ID:=6575757;',
+            '88, BNF NAME:=Some LLC; ',
+            '88, BNF ADDRESS:= Boston MA US 02116 ;',
+            '88, REC FI:=SVB;',
+            '88, REC ID:=121140399;',
+            '88, OBI:=PO 123 Inv #789',
```

Comment:
A couple of comments on this sample:

These aren’t dummy details are they? I’m not super comfortable including genuine addresses even if they are business and not individuals.

I note that there are more space separating fields, is that verbatim from your source files? If so, they’d be perfectly fine to include in these tests as it potentially increases coverage of BAI2 variations.

Reply:
I added this test a while ago.  The text was originally taken directly from a failing file we received from the Silicon Valley Bank.  I suspect I already modified what I thought was sensitive information, but will do it again and make it more obvious that the values are not real.